terraform/main.tf

```diff
@@ -1,13 +1,3 @@
-terraform {
```

Comment:
I think this should stay - it allows for planning with out having secrets locally

Reply:
so the reason i got rid of that is because I was creating dev environments with local state and I had to comment that out to do that. And since it isn't needed for tf cloud I thought I'd just get rid of it